src/connector.test.tsx

```diff
@@ -87,24 +105,6 @@ function setup() {
     dev: false,
   });
 
-  type Creator<T extends ActionTypes> = ActionCreator<Extract<Actions, { type: T }>>;
```

Comment:
Just lifting these type definitions outside of closure so that I can reference them in the SFC tests